Yosemite/Yosemite/Actions/TaxClassesAction.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import Networking
+
+
+/// TaxClassesAction: Defines all of the Actions supported by the TaxClassesStore.
+///
+public enum TaxClassesAction: Action {
+
+    /// Retrieve and synchronizes Tax Classes matching the specified criteria.
+    ///
+    case retriveTaxClasses(siteID: Int, onCompletion: ([TaxClass]?, Error?) -> Void)
```

Comment:
nit: `retrieve*` :)

Reply:
how about `Int64` for `siteID`? sorry I haven't written the p2 post yet, but I think it makes sense to follow the type in the persistent store (`Int64` instead of `Int` in some parts of the app)